content/guides/deploying-a-gitlab-cluster-on-gke/index.md

```diff
@@ -0,0 +1,541 @@
+  default     = "10.5.0.0/28"
+}
+
+# For the example, we recommend a /16 network for the VPC. Note that when changing the size of the network,
+# you will have to adjust the 'cidr_subnetwork_width_delta' in the 'vpc_network' -module accordingly.
+variable "vpc_cidr_block" {
+  description = "The IP address range of the VPC in CIDR notation. A prefix of /16 is recommended. Do not use a prefix higher than /27."
+  default     = "10.3.0.0/16"
+}
+
+# For the example, we recommend a /16 network for the secondary range. Note that when changing the size of the network,
+# you will have to adjust the 'cidr_subnetwork_width_delta' in the 'vpc_network' -module accordingly.
+variable "vpc_secondary_cidr_block" {
+  description = "The IP address range of the VPC's secondary address range in CIDR notation. A prefix of /16 is recommended. Do not use a prefix higher than /27."
+  default     = "10.4.0.0/16"
+}
+
+```
+
+**Note:** Be sure to fill in any required variables that don't have a default value.
+
+Now we can use Terraform to create the resources:
+
+1. Run `terraform init`.
+2. Run `terraform plan`.
+3. If the plan looks good, run `terraform apply`.
+
+Terraform will begin to create the GCP resources. This process can take up to XX minutes.
+
+## Installing GitLab
+
+We install GitLab by following the recommended way using a Helm chart. The chart contains all of the required components
+to run GitLab and can easily scale to large deployments.
+
+More information about the chart is explain on the GitLab cloud native Helm chart page. [GitLab cloud native Helm Chart | GitLab](https://docs.gitlab.com/charts/)
+
+**Note:** Some GitLab features are currently not available when using the Helm chart:
+
+- GitLab Pages
+- GitLab Geo
+- No in-cluster HA database
+- Smartcard authentication
+
+We can check the latest versions using the following command:
+
+```bash
+$ helm search -l gitlab/gitlab | head -n4
+NAME                    CHART VERSION   APP VERSION     DESCRIPTION
+gitlab/gitlab           1.9.0           11.11.0         Web-based Git-repository manager with wiki and issue-trac...
+gitlab/gitlab           1.8.4           11.10.4         Web-based Git-repository manager with wiki and issue-trac...
+gitlab/gitlab           1.8.3           11.10.3         Web-based Git-repository manager with wiki and issue-trac...
+```
+
+**Note**: The GitLab chart doesn't have the same version number as GitLab itself. For more information please refer to the [GitLab version mappings](https://docs.gitlab.com/charts/installation/version_mappings.html) document.
+
+Please read the GitLab [Helm Deployment Guide](https://docs.gitlab.com/charts/installation/deployment.html) for more information on customizing
+the GitLab installation.
+
+## Deploying the Dockerized App
```

Comment:
Do we need this section for this doc, when it is already covered in the other doc?

Reply:
@yorinasub17 nope its pure copy pasta and I'll remove it plus add the Helm install GitLab chart bit.